CMakeLists.txt

```diff
@@ -623,6 +623,13 @@ if (LLVM_ENABLE_DOXYGEN)
   message(STATUS "Doxygen: enabled")
 endif()
 
+if(SWIFT_ANALYZE_CODE_COVERAGE)
+  set(LIBCLANG_RT_PROFILE "${LLVM_LIBRARY_DIR}/clang/${CLANG_VERSION}/lib/darwin/libclang_rt.profile_osx.a")
```

Comment:
We can't use the just-built runtime libraries with the code built by the host compiler.  Even if this succeeds, it is pure luck and an unsupported configuration as far as I know.

We need to track use the runtime library provided with the toolchain for code built by the host clang.